src/coreclr/System.Private.CoreLib/src/System/Threading/ThreadPool.CoreCLR.cs

```diff
@@ -234,6 +234,9 @@ private static unsafe bool InitializeConfigAndDetermineUsePortableThreadPool()
                 ? AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.EnableWorkerTracking", false)
                 : GetEnableWorkerTrackingNative();
 
+        internal static readonly bool EnableDispatchAutoReleasePool =
+            AppContextConfigHelper.GetBooleanConfig("System.Threading.ThreadPool.EnableDispatchAutoreleasePool", false);
```

Comment:
Do we have a test that sets this AppContext switch to its non-default value?

Reply:
I need to update the runtime test to do that. What's the best way to do so?

Reply:
This is how I tested the `EventSource.IsSupported` AppContext switch. It uses `RemoteExecutor` to set the switch:

https://github.com/dotnet/runtime/blob/1d9e50cb4735df46d3de0cee5791e97295eaf588/src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestNotSupported.cs#L22-L49

Reply:
Is that even something we support today?

Reply:
Since the test needs native assets, I can't use RemoteExecutor since we're executing in the CoreCLR runtime tests tree.

Reply:
Looks like we need to enable feature flags in some manner for CoreRun

Reply:
Any reason you can't add a test to `src\libraries\System.Runtime\tests`?

Another thought is that you can set `AppContext` values really early in the process, if you can get your test code executed before the runtime tries reading the value.

Reply:
The test needs to have a native component to accurately test it.

I'll try setting the value early and see if that works. That's significantly easier than updating CoreRun.

Reply:
Using the AppContext API works. Thanks!